examples/nextjs/next.config.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  async headers() {
+    return [
+      {
+        source: '/(.*)',
```

Comment:
```suggestion
        source: '/:path*',
```

otherwise it will not be applied on the homepage